GeoLib/Station.cpp

```diff
@@ -23,13 +23,13 @@
 
 namespace GeoLib
 {
-Station::Station(double x, double y, double z, std::string name) :
-    Point (x,y,z), _name(std::move(name)), _type(Station::StationType::STATION),
+Station::Station(double x, double y, double z, std::string const& name) :
```

Comment:
Shouldn't interfere: It's only about the convention "pass big objects by reference".